pom.xml

```diff
@@ -105,6 +107,12 @@
                 <version>${dep.minio.version}</version>
             </dependency>
 
+            <dependency>
```

Comment:
Why is this being added? We should carefully consider every new dependency we add. Please discuss with the group before adding new dependencies.